### PR TITLE
Added Footer 11 lift blue #435

### PR DESCRIPTION
**Pull request by AmitDeka**
Issue #435 
Footer 11 blue

### CODE REVIEW

**Comment by prem-io** [comment]
@AmitDeka Just suggestion. Can you handle responsive and theme based in the same component. This will make you're component robust.

File: Footer/Footer 11 lift blue.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<footer class="bg-blue-600 antialiased">
```
Comment (prem-io) [inline Footer/Footer 11 lift blue.html:1]: To handle dark/light theme changes. You can use this.
`bg-white dark:bg-blue-600`

File: Footer/Footer 11 lift blue.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      </h1>
+      <p class="text-center font-normal text-sm leading-6 text-white/80 mb-8">
+        We’re a diverse and passionate team that takes ownership of your design
+        and empower you to execute the roadmap. We stay light on our feet and
+        truly enjoy delivering great work.
+      </p>
+      <div class="w-100 flex justify-center gap-x-8">
+        <a
+          class="inline-flex items-center bg-gray-100 py-1.5 px-4 rounded-full"
+          href=""
```
Comment (prem-io) [inline Footer/Footer 11 lift blue.html:15]: use `href="#"` for empty link redirections

**Comment by prem-io** [comment]
@AmitDeka Just suggestion. Can you handle responsive and theme based in the same component. This will make your component robust.

File: Footer/Footer 11 lift blue.html (hunk 1 above)
Comment (AmitDeka) [reply Footer/Footer 11 lift blue.html:1]: Thank you for your suggestion. I can do that too but this component has a black variant too

File: Footer/Footer 11 lift blue.html (hunk 2 above)
Comment (AmitDeka) [reply Footer/Footer 11 lift blue.html:15]: Thank you for your suggestion. I will commit these changes ASAP

**Comment by Rajdip019** [comment]
Good job @AmitDeka 

Congrats 🎉, your PR got merged. To be eligible for the swags from ClueLess. Follow this steps.

1. Star this repository. ⭐
2. SignIn / SignUp in [[ClueLess Website](https://www.clueless.tech/)](https://www.clueless.tech/) .
3. Go to profile section from the top right hand corner.



1. Comment your profile URL under this comment tagging the person who merged your PR.

That’s it. Follow the leaderboard !

And don’t forget to share your achievement with your friends.

**Comment by AmitDeka** [comment]
https://www.clueless.tech/profile/40214392 @Rajdip019

**Pull request merged**